### PR TITLE
No dns on system start

### PR DESCRIPTION
**Pull request by nm-mrt**
If there is no DNS available on system start the capwap-wtp will stop while evaluating the configuration. I fixed that by checking if the address was yet translated on every address access. If not a DNS request is started before continuing.

With this patch I get the system up on boot if DNS shows up later (here I have a cellular connection and that takes some time to show up). 

Sorry for the many commits, but I had to missuse github for deployment to my build server. Just have a look at the full diff. It's not that much.

### CODE REVIEW

File: src/wtp/wtp.c [hunk 1]
```diff
@@ -1068,20 +1069,22 @@ static int wtp_parsing_configuration_1_0(config_t* config) {
 		for (i = 0; i < count; i++) {
 			const char* address = config_setting_get_string_elem(configSetting, i);
 			if (address != NULL) {
-				union sockaddr_capwap acaddr;
+				struct addr_capwap acaddr;
+				memset(&acaddr, 0, sizeof(struct addr_capwap));
+				strncpy(acaddr.fqdn, address, CAPWAP_MAX_FQDN_SIZE-1);
```
Comment (thz) [inline src/wtp/wtp.c]: either dynamically allocate fqdn, or clearly fail if address from config is too long.
currently we have bad behaviour here for too long addresses.

File: src/wtp/wtp.c [hunk 2]
```diff
@@ -1094,19 +1097,21 @@ static int wtp_parsing_configuration_1_0(config_t* config) {
...
```
Comment (thz) [inline src/wtp/wtp.c]: handling for too long address missing (like above)

**Comment by thz** [comment]
please rebase on v1.1.x or master (the latter is preferred)

**Comment by nm-mrt** [comment]
You are absolutely correct - should have done that in the first place.

**Pull request closed without merging**